app/modules/ads/ad-context.js

```diff
@@ -450,6 +450,10 @@ export const defaultAdContext = {
       enabled: false,
       propertyId: 'd-aIf3ibf0cYxCLB1HTWfBQOFEA',
     },
+    durationMedia: {
+      enabled: false,
+      siteId: '1066',
```

Comment:
Incorrect ID, Sir. Check epic attachments, please.

Reply:
You right, changed 👍